graph/push.go

```diff
@@ -10,13 +10,19 @@ import (
 	"github.com/docker/docker/registry"
 )
 
+// ImagePushConfig stores push configuration.
 type ImagePushConfig struct {
+	//MetaHeaders store meta data about the image (DockerHeaders with prefix X-Meta- in the request)
 	MetaHeaders map[string][]string
-	AuthConfig  *cliconfig.AuthConfig
-	Tag         string
-	OutStream   io.Writer
+	//AuthConfig is to authorize the new session with the registry
+	AuthConfig *cliconfig.AuthConfig
+	//Tag is the new tag used when image is pushed
```

Comment:
Tag is the specific image to push. If no tag is provided, all tags will be pushed. <- Or something like that